Mlem/Views/Tabs/Posts/Components/Post/Components/Community View/Community View.swift

```diff
@@ -465,26 +442,34 @@ struct CommunityView: View
             .buttonStyle(EmptyButtonStyle()) // Make it so that the link doesn't mess with the styling
             .task {
                 if post == postTracker.posts.last {
-                    if postTracker.posts.isEmpty {
-                        postTracker.isLoading = true
-                    }
-
                     await loadFeed()
-                    postTracker.isLoading = false
                 }
             }
         }
     }
+    
+    @ViewBuilder
+    private var loadingMorePostsView: some View {
+        if postTracker.isLoading {
+            VStack(alignment: .center) {
+                ProgressView()
+                    .frame(width: 16, height: 16)
+                Text("Loading more posts...")
+            }
+            .padding()
+            .frame(maxWidth: .infinity)
+            .foregroundColor(.secondary)
+            .background(Color.systemBackground)
```

Comment:
can you add
```
.accessibilityElement(children: .combine)
```

Reply:
added here -> fae8e187d30724b3f70e5a80b6c213c1ec1176cd